web/apps/labelstudio/src/pages/CreateProject/Config/Config.jsx

```diff
@@ -40,36 +41,28 @@ const Label = ({ label, template, color }) => {
 
   return (
     <li className={configClass.elem("label").mod({ choice: label.tagName === "Choice" })}>
-      <label style={{ background: color }}>
-        <Input
-          type="color"
-          className={configClass.elem("label-color")}
-          value={colorNames[color] || color}
-          onChange={(e) => template.changeLabel(label, { background: e.target.value })}
-        />
-      </label>
-      <span>{value}</span>
-      <button
+      <span className="flex">
+        <label style={{ background: color }}>
+          <input
+            type="color"
+            className={configClass.elem("label-color")}
+            value={colorNames[color] || color}
+            onChange={(e) => template.changeLabel(label, { background: e.target.value })}
+          />
+        </label>
+        <span>{value}</span>
+      </span>
+      <Button
         type="button"
-        className={configClass.elem("delete-label")}
+        look="string"
+        size="smaller"
+        variant="neutral"
```

Comment:
i see variant="neutral" is set here, but icon is a trash icon, should this be `variant="negative"` instead?

Reply:
good catch!